lang/sensei-lms.pot

```diff
@@ -1,5 +1,5 @@
 # Copyright (C) 2021 Automattic
-# This file is distributed under the same license as the Sensei LMS plugin.
+# This file is distributed under the GPL version 2 or later - http://www.gnu.org/licenses/old-licenses/gpl-2.0.html.
```

Comment:
It's not necessary to commit this file since we rebuild it once as part of the plugin release process. But it doesn't hurt either. 🙂

Reply:
Committed it since I've used it as testing proof, can uncommit it ofc :D